src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -133,6 +133,65 @@ public void testLegacyAccountEntityToAttributes() {
 
     }
 
+    @Test
+    public void getCopy_shouldCreateDeepCopy() {
+        String accountGoogleId = "johnDoe";
+
+        AccountAttributes account = AccountAttributes.builder()
+                .withGoogleId(accountGoogleId)
+                .withName("John")
+                .withEmail("johnDoeEmail")
+                .withInstitute("johnson")
+                .withIsInstructor(true)
+                .build();
+        account.isInstructor = true;
```

Comment:
There is no need to assign `true` again since you already pass `.withIsInstructor(true)` to the builder